plugins/modules/aci_contract_subject.py

```diff
@@ -51,6 +57,34 @@
     choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43,
                CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
     aliases: [ target ]
+  priority_consumer_to_provider:
+    description:
+    - The QoS class of Filter Chain For Consumer to Provider.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ level1, level2, level3, unspecified ]
+  dscp_consumer_to_provider:
+    description:
+    - The target DSCP of Filter Chain For Consumer to Provider.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43,
+               CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
+    aliases: [ target ]
+  priority_provider_to_consumer:
+    description:
+    - The QoS class of Filter Chain For Provider to Consumer.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ level1, level2, level3, unspecified ]
+  dscp_provider_to_consumer:
+    description:
+    - The target DSCP of Filter Chain For Provider to Consumer.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43,
+               CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
+    aliases: [ target ]
```

Comment:
dscp_provider_to_consumer, dscp_consumer_to_provider -> both the parameters have same aliases: [ target ]

Reply:
ok. regular dscp also which I also changed